ultratenant/threadlocal.py

```diff
@@ -0,0 +1,25 @@
+import threading
+
+
+class TenantLocal(threading.local):
+    UNDEFINED = ""
+
+    def __init__(self):
+        self._tenant = self.UNDEFINED
+
+    @property
+    def tenant(self):
+        return self._tenant
+
+    @tenant.setter
+    def tenant(self, name):
+        self._tenant = name
```

Comment:
Why use property and setter instead of just use the variable? Example:

```suggestion
        self.tenant = self.UNDEFINED
```